Code/GraphMol/Depictor/RDDepictor.cpp

```diff
@@ -315,7 +317,15 @@ unsigned int compute2DCoords(RDKit::ROMol &mol,
     if (coordMap) {
       params.coordMap = *coordMap;
     }
-    return RDKit::CoordGen::addCoords(mol, &params);
+    unsigned int cid = RDKit::CoordGen::addCoords(mol, &params);
+    RDGeom::Point3D centroid = MolTransforms::computeCentroid(mol.getConformer(cid));
```

Comment:
We shouldn't be calculating the centroid and doing the coordinate shift if `coordMap` is provided.